Makefile

```diff
@@ -1,7 +1,7 @@
 export CARGO_PROFILE_RELEASE_CODEGEN_UNITS = 1
 export CARGO_PROFILE_RELEASE_LTO = fat
 export DOCKER_BUILDKIT = 1
-export RUSTFLAGS = "-D warnings -C target-cpu=native"
+export RUSTFLAGS = "-D warnings -C target-feature=+sse2 -C target-feature=+sse4.1 -C target-feature=+sse4.2"
```

Comment:
you can avoid some of the duplication by

```
-Ctarget-feature=+sse2,+sse4.1,+sse4
```

Additionally `sse2` is part of the baseline `x86_64` target already, so there is no need to specify it.

Reply:
Is there a way we could specify this in Cargo.toml or some place like that? While our release binaries use Makefile and our instructions for validators are to build with Makefile it would be nice to have faster code during testing and development as well.

Reply:
This line isn’t necessary since we have .cargo/config.toml, is it?